<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Panel administratora</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj nową osobę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_teach">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Naucz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_show">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Rozpoznaj twarze</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
